src/components/VHeader/VSearchBar/VSearchButton.vue

```diff
@@ -8,7 +8,7 @@
     :class="[
       isIcon
         ? 'search-button focus-visible:bg-pink focus-visible:text-white p-[0.5px] ps-[1.5px]'
-        : 'py-6 px-10 h-full',
+        : 'py-6 px-6 whitespace-nowrap h-full',
```

Comment:
Let's keep the horizontal padding to `px-10` as it is in the [Figma mockup](https://www.figma.com/file/w60dl1XPUvSaRncv1Utmnb/Audio-Release?node-id=1424%3A20573).
```suggestion
        : 'py-6 px-10 whitespace-nowrap h-full',
```